hello.lua

```diff
@@ -10,11 +12,12 @@ local config = {
 local client = ld.clientInit(config, 1000)
 
 local user = ld.makeUser({
-    key = "abc"
+    key = "example-user-key",
+    name = "Sandy"
 })
 
 if client:boolVariation(user, YOUR_FEATURE_KEY, false) then
-    print "feature is enabled"
+    print "feature flag is true for this user"
```

Comment:
I wasn't able to find an example of including a variable in a message, but ideally this would say "feature flag [flag key] is true for this user"